src/fileshelter/ui/FileShelterApplication.hpp

```diff
@@ -46,7 +47,10 @@ namespace UserInterface
 			void notify(const Wt::WEvent& event) override;
 
 			void displayError(std::string_view error);
-
+            
+            Wt::WMenuItem* menuItemShareCreate;
```

Comment:
please make it private + prefix with _

Reply:
It is private (in hpp - find it was needed in there so that the other methods can access it but will add _.